content/docs/tasks/telemetry/servicegraph/index.md

```diff
@@ -5,6 +5,8 @@ weight: 50
 keywords: [telemetry,visualization]
 ---
 
+> {{< warning_icon >}} The Service Graph is being deprecated in Istio 1.1 in favor of [Kiali](../kiali).
```

Comment:
The note, IMO is not enough. If the Istio service graphs are to be deprecated, the content should be archived and unpublished. @sdake @geeknoid Is there a content deprecation process?

Reply:
@jeffmendoza @sdake @douglas-reid  - correct me if I'm wrong in what I say below. In short, Service Graph add-on and its docs are not to be removed yet (only marked as deprecated).

As I understand it, the Service Graph add-on is not being removed - just marked as deprecated for removal in a later Istio version. So the Service Graph will still exist and ship with Istio 1.1. Thus, it was my understanding the Service Graph docs (such as the Task and its README) will remain but will have a deprecation notice. Those docs will then be removed when a future Istio version (1.2 probably) is released.

In fact, see this [Istio PR #9094](https://github.com/istio/istio/pull/9094) for the deprecation notice added to the README (which was to be done in tandem with this change in the Service Graph Task that you see here).

Reply:
Not sure about docs. Service Graph code certainly needs to stay in istio/istio, and installable via helm option, not removed or archived.

I can see the case for removing docs from the site. We don't want new users to discover and start using it. I'll defer to @geeknoid for a decision.